auto-value-parcel/src/test/java/com/ryanharter/auto/value/parcel/model/SampleTypeWithParcelableContractSatisfied.java

```diff
@@ -0,0 +1,28 @@
+package com.ryanharter.auto.value.parcel.model;
```

Comment:
Not sure if you want this class moved elsewhere. This class has to have a static member which isn't allowed using the existing standard of having inner classes within AutoValueParcelExtensionTest.